web/src/components/organisms/Project/ModelsMenu/hooks.ts

```diff
@@ -0,0 +1,138 @@
+import { useCallback, useEffect, useMemo, useState } from "react";
+
+import { Model } from "@reearth-cms/components/molecules/Schema/types";
+import {
+  useGetModelsQuery,
+  useCreateModelMutation,
+  useCheckModelKeyAvailabilityLazyQuery,
+} from "@reearth-cms/gql/graphql-client-api";
+
+type Params = {
+  projectId?: string;
+  modelId?: string;
+};
+
+export default ({ projectId, modelId }: Params) => {
+  const [modelModalShown, setModelModalShown] = useState(false);
```

Comment:
```suggestion
  const [modelModalVisibility, setModelModalVisibility] = useState(false);
```
a suggestion?

Reply:
@nourbalaha , I think they are pretty similar.
Inoue-san used both `xxxModalVisible` and `xxxModalShown` in re:earth codebase
let's see what @KaWaite thinks.

Reply:
Grammatically speaking, it should be `xxVisible` instead of `xxVisibility`.
But both are pretty valid. I tend to use xxShow or xxShown because they are shorter, but yeah whatever you guys want since they both convey the meaning well.